error/AppErrors.ts

```diff
@@ -0,0 +1,13 @@
+const i18n = require('../loc');
+
+export class DoubleSpentFundsError extends Error {
```

Comment:
Not sure if it's best way to create custom errors. This way we can't assign custom error name. Not sure if it's a big difference, but I'm doing it this way:
```
class ValidationError extends Error {
  constructor(message) {
    super(message);
    this.name = "ValidationError";
  }
}
```

Up to you, if you don't see any benefits of this approach then let's skip it

Reply:
Added a possibility to provide own message